src/lambda_functions/get_prices_lambda/scraper.py

```
@@ -4,6 +4,7 @@
 import os
 import re
 import time
+import uuid
```

Ruff (0.8.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)